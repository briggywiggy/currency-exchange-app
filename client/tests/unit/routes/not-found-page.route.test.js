import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../src/routes/NotFoundPage/not-found-page.route';

test('should render LoadingPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})