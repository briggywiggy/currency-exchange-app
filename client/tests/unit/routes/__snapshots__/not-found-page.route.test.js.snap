// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="not-found-page page"
>
  404! - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
